### PR TITLE
alanlima/c02-network04

### PR DESCRIPTION
**Pull request by alanlima**
# Description

Exercise (e.g. C01-E01): c02-network04

Student Name: Alan Lima

> A brief explanation of what has been done.

## Type of change

- [x] Exercise Submission (one exercise per PR)
  - [x] This PR is just for **one** exercise of a class
  - [x] PR name follows the pattern `<github-username>/<exercise-number>`
  - [x] Added exercise files inside folder `/classes/<class- number>/exercises/<exercise-number>/<github-username>/`
	- [x] Review [exercise submission steps](./README.md#Exercises)
- [ ] Class content (instructors/admins)
- [ ] Documentation update
- [ ] Repo management

### CODE REVIEW

File: classes/02class/exercises/c02-network04/alanlima/README.md [hunk 1]
```diff
@@ -0,0 +1,204 @@
...
+## Commands Execution Output
+
+- Commands for creating the subnets:
+```
+# Create VPC
+
+$ aws ec2 create-vpc \
+    --cidr-block 172.16.0.0/16 \
+    --no-amazon-provided-ipv6-cidr-block \
+    --instance-tenancy default
```
Comment (kikobr82) [inline classes/02class/exercises/c02-network04/alanlima/README.md:12]: isn't your VPC created already in lab03?

File: classes/02class/exercises/c02-network04/alanlima/README.md [hunk 2]
```diff
@@ -0,0 +1,204 @@
...
+AssociationId: rtbassoc-009db4028ee49bcc7
+AssociationState:
+  State: associated
+
+# Modify the subnets to map public ip for new nodes inside the public subnets
+$ aws ec2 describe-subnets \
+    --filters "Name=tag:Name,Values=public-a,public-b,public-c" \
+    --output text \
+    --query 'Subnets[*].{Id:SubnetId}' \
+    | xargs -I % sh -c "echo Modifying subnet to %; aws ec2 modify-subnet-attribute --subnet-id % --map-public-ip-on-launch"
```
Comment (kikobr82) [inline classes/02class/exercises/c02-network04/alanlima/README.md:183]: this seems to be part of future labs...

File: classes/02class/exercises/c02-network04/alanlima/scripts.sh [hunk 3]
```diff
@@ -0,0 +1,196 @@
+#!/bin/bash
```
Comment (kikobr82) [inline classes/02class/exercises/c02-network04/alanlima/scripts.sh:1]: Nice script, but you'll see how this won't make a lot of sense during class 04..:)

File: classes/02class/exercises/c02-network04/alanlima/README.md (hunk 1 above)
Comment (alanlima) [reply classes/02class/exercises/c02-network04/alanlima/README.md:12]: I usually delete all resources once I finished the lab. so I needed to create it again. 😅😅

File: classes/02class/exercises/c02-network04/alanlima/README.md (hunk 2 above)
Comment (alanlima) [reply classes/02class/exercises/c02-network04/alanlima/README.md:183]: Yeah, once that I started to read the following exercises I see that I already did.
I will send a new commit removing those extra scripts.

File: classes/02class/exercises/c02-network04/alanlima/scripts.sh (hunk 3 above)
Comment (alanlima) [reply classes/02class/exercises/c02-network04/alanlima/scripts.sh:1]: Thanks, I can see that. Creating bash scripts is quite good to learn the commands and for quick tests. But I can see a lot of problems in real case scenarios. 
I am looking forward to learning more about terraform ;)

**Pull request merged**